main.go

```diff
@@ -76,7 +77,8 @@ var (
 )
 
 type prometheus struct {
-	ruleManager         manager.RuleManager
+	queryEngine         *promql.Engine
+	ruleManager         rules.RuleManager
```

Comment:
Doesn't necessarily have to be in this PR, but it'd be nice to rename RuleManager -> Manager, since it's already in the rules package anyways (Go convention).